ui/qtgalacticplot.py

```diff
@@ -54,10 +55,20 @@ def plotGalaxy(self, planets, tradeRoutes, allPlanets, planetOwners = [], autoPl
         self.__annotate.set_visible(False)
 
         self.__planetNames = []
+        self.__planetOwners = []
 
         x = []
         y = []
 
+        for ap in allPlanets:
+            found_pa = False
+            for pa, po in zip(planets, planetOwners):
+                if ap.name == pa.name:
+                    self.__planetOwners.append(po.name)
+                    found_pa = True
+            if found_pa == False:
+                self.__planetOwners.append("")
```

Comment:
Couldn't this just be part of the loop that colours the planets? Down at line 99 (110 in your edits)

Reply:
The loop that colours the planets only handles the subset of planets that are **used** in the GC (i.e. `planets` and `planetOwners` variables passed in from plotGalaxy) whereas the annotation is based on `allPlanets` which includes planets not used in the GC. When I tried to use just `planets` and `planetOwners` bad things happened at the annotation because the array lengths are different and the _n_ indexes are misaligned.